WooCommerce/Classes/ViewRelated/Dashboard/JetpackInstall/JetpackInstallStepsViewModel.swift

```diff
@@ -17,40 +17,75 @@ final class JetpackInstallStepsViewModel: ObservableObject {
     ///
     @Published private(set) var currentStep: JetpackInstallStep?
 
+    /// Whether the install failed. This will be observed by `JetpackInstallStepsView` to present error modal.
+    ///
+    @Published var installFailed: Bool = false
```

Comment:
nit: this could also be `private(set)`?
```suggestion
    @Published private(set) var installFailed: Bool = false
```

Reply:
Thanks! Fixed in 95c4ebf97b6cd34c93238b4a89638b8c8da31709.